backend/sass/common.blocks/_setup.scss

```diff
@@ -154,94 +176,94 @@ $kadena-orange: #F3AB3C;
   }
 }
 
-.wallet__checkbox-wrapper {
+.setup__checkbox-wrapper {
   margin: 1rem;
   .checkbox {
     color: $confirm-color;
   }
 }
 
-.wallet__terms-conditions-checkbox {
+.setup__terms-conditions-checkbox {
   color: $confirm-color;
 }
 
-.wallet__key {
+.setup__key {
   overflow-x: hidden;
   text-overflow: ellipsis;
 }
 
-.wallet__key_hidden {
+.setup__key_hidden {
   text-overflow: clip;
 }
 
-.wallet__key_type_private.wallet__key_hidden {
+.setup__key_type_private.setup__key_hidden {
   cursor: alias;
 }
 
-.wallet__delete {
+.setup__delete {
   text-align: right;
 }
 
-.wallet__add {
-  @extend .wallet__delete;
+.setup__add {
+  @extend .setup__delete;
 }
 
-.wallet__add-delete-button {
+.setup__add-delete-button {
   width: 30px;
 }
 
-.wallet__record-phrase-msg {
+.setup__record-phrase-msg {
   margin-bottom: 2em;
 }
 
-.wallet__passphrase-widget-wrapper {
+.setup__passphrase-widget-wrapper {
   display: grid;
   grid-template-columns: repeat(4, 1fr);
 }
 
-.wallet__passphrase-widget-word-hider {
+.setup__passphrase-widget-word-hider {
   background-color: $fullscreen-wallet-grey;
 }
 
-.wallet__passphrase-widget-word,
-.wallet__passphrase-widget-word-hider,
-.wallet__passphrase-widget-elem-wrapper {
+.setup__passphrase-widget-word,
+.setup__passphrase-widget-word-hider,
+.setup__passphrase-widget-elem-wrapper {
   text-align: center;
   margin-bottom:0.5em;
   font-size: 20px;
 }
 
-.wallet__passphrase-widget-word {
+.setup__passphrase-widget-word {
   display:none;
 }
 
-.wallet__passphrase-widget-elem-wrapper {
-  &:hover .wallet__passphrase-widget-word {
+.setup__passphrase-widget-elem-wrapper {
+  &:hover .setup__passphrase-widget-word {
     display: initial;
   }
-  &:hover .wallet__passphrase-widget-word-hider {
+  &:hover .setup__passphrase-widget-word-hider {
     display: none;
   }
 }
 
-.fullscreen .wallet__verify-phrase-msg {
+.setup__fullscreen .setup__verify-phrase-msg {
   margin-bottom: 1.5rem;
 }
 
-.fullscreen .wallet__message {
+.setup__fullscreen .setup__message {
```

Comment:
There are some good reasons for nested selectors, but most of the time you would want to avoid them.